From 851bd018ee92dea0014b13a8535e47697d69ac41 Mon Sep 17 00:00:00 2001
From: mogilisx <rohinix.mogilisetty@intel.com>
Date: Mon, 14 Oct 2019 17:35:32 +0530
Subject: [PATCH] OAM-86918 - CTS Test Case issue fix

Tracked-On:(OAM-86918)

(OAM-86918) [Q-MR0-CELADON-MASTER][APL-IVI-Celadon-Q][CTS]

Signed-off-by: mogilisx <rohinix.mogilisetty@intel.com>
---
 automotive/vehicle/2.0/default/impl/vhal_v2_0/DefaultConfig.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/automotive/vehicle/2.0/default/impl/vhal_v2_0/DefaultConfig.h b/automotive/vehicle/2.0/default/impl/vhal_v2_0/DefaultConfig.h
index a46de24..1d7a953 100644
--- a/automotive/vehicle/2.0/default/impl/vhal_v2_0/DefaultConfig.h
+++ b/automotive/vehicle/2.0/default/impl/vhal_v2_0/DefaultConfig.h
@@ -234,7 +234,7 @@ const ConfigDeclaration kVehicleProperties[]{
          {
              .prop = toInt(VehicleProperty::PERF_VEHICLE_SPEED),
              .access = VehiclePropertyAccess::READ,
-             .changeMode = VehiclePropertyChangeMode::ON_CHANGE,
+             .changeMode = VehiclePropertyChangeMode::CONTINUOUS,
              .minSampleRate = 1.0f,
              .maxSampleRate = 10.0f,
          },
-- 
2.7.4

